src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstiertodeliusupdate/listeners/HMPPSTierListener.kt

```diff
@@ -27,14 +29,19 @@ class HMPPSTierListener(
   @SqsListener(value = ["\${sqs.queue}"], deletionPolicy = ON_SUCCESS)
   fun onRegisterChange(message: String) {
     val sqsMessage: SQSMessage = gson.fromJson(message, SQSMessage::class.java)
+
     log.info("Received message ${sqsMessage.MessageId}")
-    val changeEvent: TierChangeEvent = gson.fromJson(sqsMessage.Message, TierChangeEvent::class.java)
-    when (changeEvent.eventType) {
-      EventType.HMPPS_TIER_CALCULATION_COMPLETE -> updateTier(crn = changeEvent.crn, calculationId = changeEvent.calculationId)
-      else -> {
-        telemetryService.invalidMessage(sqsMessage.MessageId)
-        throw InvalidMessageException("Received a message I wasn't expecting $changeEvent")
+    if (enableUpdates) {
```

Comment:
I still see this being removed/never disabled when we go properly into production, so this just a toggle for us. 
I guess we need to decide how much of this call we want to do for our benefit, we can still make the call to hmpps-tier and then not send it to delius or leave it as it is.

Reply:
Yes, the toggle should be removed in May.It's unlikely we'll make a lot of changes to this service between now and then so I don't suppose it matters much where the toggle is in the code. I would normally put it at the last possible point so we can see that everythig is going to work but this app does so little and will be working in other environments that this is probably not such an important consideration.